lib/client.js

```diff
@@ -119,11 +132,6 @@ class ProtomuxRpcConnection extends SuspendResource {
         await this._backoff.run()
       }
     }
-
-    if (this.closing || this.suspended || this.shouldBeSuspended) return
```

Comment:
This code causes reconnect spam, because it short-circuits the backoffs.
It's not needed: we'll try to connect again anyway with the next request. All it does is instantly-reconnect in the background every time the server destroys the connection (so the server can't disconnect from us if it's overloaded)